streamer/src/nonblocking/quic.rs

```diff
@@ -258,6 +258,9 @@ where
     let quic_server_params = Arc::new(quic_server_params);
     let rate_limiter = Arc::new(ConnectionRateLimiter::new(
         quic_server_params.max_connections_per_ipaddr_per_min,
+        // allow for 10x burst to make sure we can accommodate legitimate
```

Comment:
Should we do this only to test? Would this make our rate limiting ineffective against abusive clients?